src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -54,12 +55,9 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        if (dateParams.length < 3) {
-            return Optional.empty();
-        }
         try {
             return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
-        } catch (DateTimeException message) {
+        } catch (RuntimeException message) {
             return Optional.empty();
         }
```

Comment:
catching any RuntimeException is too ambiguous 
we can catch two particular exceptions in the same catch block, but in case of any trouble we should be able to identify them - for now you can use `System.out.println()`, later we will learn logging
move `return Optional.empty();` to the end of method

Reply:
done